### DISCUSSION TITLE
2024-06-21 悠长的一周

### DISCUSSION THREAD

**Discussion by giscus** [ThaddeusJiang/thaddeusjiang.github.io#73] [Announcements]
# 2024-06-21 悠长的一周

自大学毕业后来日本工作，我和父母一起生活的时间非常少，2016年春节回国待了2周，2018年夏季回国待了2周，之后遭遇疫情，我5年未和父母团聚，2023年春季回国见父母、办婚礼、在东北一周时间，忙婚庆、忙酒席、见长辈，和父母一起的时间其实非常短。我感觉和父亲的感情都陌生了，我一直觉得自己亏欠父母的养育之恩。

https://thaddeusjiang.com/2024-06-21-you-chang-de-yi-zhou

**Comment by defclass** [comment]
温暖人心的文字 。

**Comment by tcstory** [comment]
谢谢分享
